src/egon/data/datasets/etrago_setup.py

```diff
@@ -287,8 +383,30 @@ class EgonPfHvStorageTimeseries(Base):
 
 
 class EgonPfHvStore(Base):
+    source_dict = sources()
+    source_list = [
+        source_dict["bgr_inspee"],
+        source_dict["bgr_inspeeds"],
+        source_dict["bgr_inspeeds_data_bundle"],
+        source_dict["bgr_inspeeds_data_bundle"],
+        source_dict["bgr_inspeeds_report"],
+    ]
+    contributors_dict = contributors()
+    contributor_list = [
+        {key: value for key, value in contributors_dict[author]}
```

Comment:
This line seems to be problematic, somehow 'contributors_dict' is not defined in this line. But I don't understand the problem.

Reply:
This is solved in https://github.com/openego/eGon-data/pull/568/commits/0277794b7d63ecd06d78d469f942a8b08ba8ab53.